Formula/caddy.rb

```diff
@@ -13,7 +13,7 @@ class Caddy < Formula
     sha256 "cb7a32a7d3d39a332038b2724b0a70475dcb9f47c9359ce2a9e57a66849a6c21" => :high_sierra
   end
 
-  depends_on "go" => :build
+  depends_on "go@1.15" => :build
```

Comment:
This should just be `go`. When go 1.16 comes out we'll test the formula and if it doesn't work we'll pin it to 1.15.

Reply:
Why not just let us tell you when it's good to use?

The version was pinned to use Go 1.14 earlier because there was a breakage in a dependency of Caddy in relation to Go 1.15, so that dependency had to be upgraded before it Go 1.15 would work.

There's no guarantee that the same won't happen for 1.16. Pinning is the safer bet. Let us first validate in the upstream repo when we're ready to have builds use the latest version of Go, then brew can be updated to the latest.

Reply:
Because homebrew is an evergreen package manager. We always use the latest dependencies that work and since your software is open source the Homebrew project can decide to package it whatever way it likes.

Reply:
The problem is that we're responsible as maintainers of the project when things break. Users will come to us to complain.

This entire situation happened because changes were made here without getting our input, and we heard about the breakage via our forums.

I'd be much happier if things were played safe here, and the software was built in a predictable way, consistent with how it's built everywhere else.

Reply:
You can make the software closed source and have full control over the way you distribute it. Until then you'll have to deal with the best practices that package managers are using for themselves.

Some information about Homebrew's position on upstream control of packages: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap#official-vendor-taps

Reply:
I'm fine with this being just `go`. Updating...